### DISCUSSION TITLE
which tflite file to use?

### DISCUSSION THREAD

**Discussion by TungstenE2** [jomjol/AI-on-the-edge-device#753] [General]
In firmware folder I see 3 tflite files? Which to use and to configure?

![grafik](https://user-images.githubusercontent.com/39955685/161424682-e3d9030d-60e8-469b-93d2-dc86716a13b2.png)

![grafik](https://user-images.githubusercontent.com/39955685/161424712-eb953c5c-d47f-4cd5-bbcb-ba844456b13c.png)

![grafik](https://user-images.githubusercontent.com/39955685/161424748-c45ee9fd-6fac-4d96-9b77-358e1a0cfdbb.png)

**Comment by jomjol** [comment]
Basically there are two types:

- ana... --> for analog pointes
- dig... --> for digits

They must not be mixed.

But within this all should work independly. The higher the version, the more images are used for training. But this does not mean always a better result, because sometimes the older specificly for only a few trained examples are better. You can just test and try, which one works best for you.
